go.mod

```diff
@@ -19,22 +19,22 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Masterminds/semver v1.5.0
 	github.com/VividCortex/mysqlerr v0.0.0-20170204212430-6c6b55f8796f
```

Comment:
There's a lot going on on these dependencies - can we please create a separate PR for the Prometheus upgrade? I think we have many components that depend on this and it needs to be looked at in-depth. Then, we can rebase this PR against main once we've merged that.

What do you think @santihernandezc?